### PR TITLE
Use NodePort instead Port, firewalls install firewall.

### PR DESCRIPTION
**Pull request by gongysh2004**
Fixes issue at https://github.com/GoogleCloudPlatform/kubernetes/issues/9566

### CODE REVIEW

**Comment by gongysh2004** [comment]
I signed it!

**Comment by gongysh2004** [comment]
how to make it shippable? thanks

File: pkg/kubectl/cmd/create.go [hunk 1]
```diff
@@ -146,7 +146,7 @@ func printObjectSpecificMessage(obj runtime.Object, out io.Writer) {
 func makePortsString(ports []api.ServicePort) string {
 	pieces := make([]string, len(ports))
 	for ix := range ports {
-		pieces[ix] = fmt.Sprintf("%s:%d", strings.ToLower(string(ports[ix].Protocol)), ports[ix].Port)
+		pieces[ix] = fmt.Sprintf("%s:%d", strings.ToLower(string(ports[ix].Protocol)), ports[ix].NodePort)
```
Comment (thockin) [inline pkg/kubectl/cmd/create.go]: This is right for one caller of this function but not for LoadBalancers.  Probably need to pass a function pointer that takes a *api.ServicePort and returns either Port or Node Port

**Comment by brendandburns** [comment]
Shippable output:

```
--- FAIL: TestMakePortsString (0.00s)
	create_test.go:204: expected: tcp:80, saw: tcp:0.
	create_test.go:204: expected: tcp:80,udp:8080,tcp:9000, saw: tcp:0,udp:0,tcp:0.
filenames: [../../../examples/guestbook/redis-master-controller.json ../../../examples/guestbook/frontend-service.json]
W0610 10:22:27.761337   10960 cmd.go:136] -p POD is DEPRECATED and will be removed in a future version. Use exec POD instead.
W0610 10:22:27.761556   10960 cmd.go:136] -p POD is DEPRECATED and will be removed in a future version. Use exec POD instead.
W0610 10:22:27.761622   10960 cmd.go:136] -p POD is DEPRECATED and will be removed in a future version. Use exec POD instead.
W0610 10:22:27.762072   10960 cmd.go:136] -p POD is DEPRECATED and will be removed in a future version. Use exec POD instead.
W0610 10:22:27.764182   10960 cmd.go:136] -p POD is DEPRECATED and will be removed in a future version. Use exec POD instead.
W0610 10:22:27.770265   10960 cmd.go:136] -p POD is DEPRECATED and will be removed in a future version. Use exec POD instead.
W0610 10:22:27.773092   10960 cmd.go:136] -p POD is DEPRECATED and will be removed in a future version. Use exec POD instead.
FAIL
```

**Comment by brendandburns** [comment]
Obsoleted by https://github.com/GoogleCloudPlatform/kubernetes/pull/10134

**Pull request closed without merging**